docs/Safety.md

```diff
@@ -4,14 +4,22 @@ As many can attest, multirotors and RC models in general can be very dangerous,
 * **NEVER** arm your model with propellers fitted unless you intend to fly!
 * **Always** remove your propellers if you are setting up for the first time, flashing firmware, or if in any doubt.
 
-## Before installing the Flight Controller in your craft
+## Before Installing
 
-Please consult the Cli.md, Controls.md, Failsafe.md and Modes.md, documentations for further important information and familiarisation with CleanFlight's terminolgies.
+Please consult the [Cli](Cli.md), [Controls](Controls.md), [Failsafe](Failsafe.md) and [Modes](Modes.md) 
+pages for further important information.
 
-You are highly advised to use the Receiver tab in CleanFlight Configurator, making sure your Rx channel values are centered at 1500 (1520 for Futaba RC) and Min/Max values are reached when controls are operated.
-The referenced values for each channel, have marked impact on the operation of Flight Controller and its Flight Modes.
+You are highly advised to use the Receiver tab in the CleanFlight Configurator, making sure your Rx channel 
+values are centered at 1500 (1520 for Futaba RC) and Min/Max values are reached when controls are operated.
```

Comment:
I think it should be explicitly stated that the min/max values should be 1000 and 2000, since a lot of people struggle with this and can't work out why they can't arm (endpoints not far enough) or failsafe immediately activates (endpoints too far).